android/src/main/java/com/imagepicker/utils/MediaUtils.java

```diff
@@ -66,6 +66,35 @@
         return result;
     }
 
+    public static @Nullable File createNewVideoFile(@NonNull final Context reactContext,
+                                                    @NonNull final ReadableMap options,
+                                                    @NonNull final boolean forceLocal)
+    {
+        final String filename = new StringBuilder("video-")
+                .append(UUID.randomUUID().toString())
+                .append(".mp4")
```

Comment:
I'm assuming that most video files will be mp4, like `createNewFile` assumes most photo files will be jpg.